[Cases] Flaky/Broken Add/Draft Comment Tests

jcger opened the pull request:
hopefully fixes https://github.com/elastic/kibana/issues/169875
hopefully fixes https://github.com/elastic/kibana/issues/169876

closes https://github.com/elastic/kibana/issues/168505
closes https://github.com/elastic/kibana/issues/168506
closes https://github.com/elastic/kibana/issues/168507
closes https://github.com/elastic/kibana/issues/168508
closes https://github.com/elastic/kibana/issues/168509

elasticmachine commented:
Pinging @elastic/response-ops (Team:ResponseOps)

elasticmachine commented:
Pinging @elastic/response-ops-cases (Feature:Cases)

adcoelho reviewed (approved):
There are two different issues for `addComment` so It seems we both touched the same test in our PRs. I just moved some logic to the e2e tests in mine(like checking the text area is empty which was missing there).

I like your solution, I thought that `act` on `onSuccess` before was a bit weird.

Let's see if this fixes things 😁

jcger commented:
> There are two different issues for `addComment` so It seems we both touched the same test in our PRs. I just moved some logic to the e2e tests in mine(like checking the text area is empty which was missing there).
> 
> I like your solution, I thought that `act` on `onSuccess` before was a bit weird.
> 
> Let's see if this fixes things 😁



> There are two different issues for `addComment` so It seems we both touched the same test in our PRs. I just moved some logic to the e2e tests in mine(like checking the text area is empty which was missing there).
> 
> I like your solution, I thought that `act` on `onSuccess` before was a bit weird.
> 
> Let's see if this fixes things 😁

I'm not sure it will fix the flakyness. If not, we can go ahead with your option
